src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -260,6 +260,52 @@ emit_xcompare (MonoCompile *cfg, MonoClass *klass, MonoTypeEnum etype, MonoInst
 	return ins;
 }
 
+static MonoInst*
+emit_xcompare_for_intrinsic (MonoCompile *cfg, MonoClass *klass, int intrinsic_id, MonoTypeEnum etype, MonoInst *arg1, MonoInst *arg2)
+{
+	MonoInst *ins = emit_xcompare (cfg, klass, etype, arg1, arg2);
+
+	gboolean is_unsigned;
+	switch (etype) {
+	case MONO_TYPE_U1:
+	case MONO_TYPE_U2:
+	case MONO_TYPE_U4:
+	case MONO_TYPE_U8:
```

Comment:
Does mono not need to consider `unsigned native int` here (UIntPtr)? Is there a `MONO_TYPE_U` or existing helper to determine signedness to help avoid breaks when new types are added in the future?

Reply:
I forgot to include `MONO_TYPE_U`. It's fixed now.